websites/T/Tinkercad/dashboard.ts

```diff
@@ -0,0 +1,79 @@
+import { Assets } from "./presence";
+
+const enum SubPages {
+	dashboard = "",
+	classes = "classes",
+	designs = "designs",
+	collections = "collections",
+	tutorials = "tutorials",
+	challenges = "challenges",
+	bin = "bin",
+}
+
+const handleDesigns = (
+	subsubpage: string,
+	presenceData: PresenceData
+): void => {
+	switch (subsubpage) {
+		case "3d": {
+			presenceData.details = "viewing 3D designs";
+			presenceData.smallImageKey = Assets.ThreeDIco;
+			presenceData.smallImageText = "3D";
+			break;
+		}
+		case "circuits": {
+			presenceData.details = "viewing circuits";
+			presenceData.smallImageKey = Assets.CircuitIco;
+			presenceData.smallImageText = "circuit";
+			break;
+		}
+		case "codeblocks": {
+			presenceData.details = "viewing codeblocks";
+			presenceData.smallImageKey = Assets.CodeblockIco;
+			presenceData.smallImageText = "code";
+			break;
+		}
+		case "bin": {
+			presenceData.details = "viewing Trash Can";
+			presenceData.smallImageKey = Assets.TrashCanIco;
+			presenceData.smallImageText = "trash";
+			break;
```

Comment:
These details should be capitalized to be consistent with the rest of the presence